jax/_src/cudnn/fused_attention_stablehlo.py

```diff
@@ -751,40 +846,51 @@ def dot_product_attention(query: Array,
                           is_causal_mask: bool = False,
                           seed: int = 42,
                           dropout_rate: float = 0.,
+                          qkv_layout: str = 'BTNH',
                           is_training = False):
-  """Computes dot-product attention given query, key, and value.
-  This is the core function for applying attention based on
-  https://arxiv.org/abs/1706.03762. It calculates the attention weights given
-  query and key and combines the values using the attention weights.
-  batch seq num_heads, head_dim // but all assume Q, K and V will have same
-  b q_seq num_heads head_dim  -> Q
-  b kv_seq num_heads head_dim -> K
-  b kv_seq num_heads head_dim -> V
+  """Computes dot-product attention given query (Q), key (K), and value (V).
+
+  This function serves as the core operation for applying attention
+  mechanisms as described in the paper [https://arxiv.org/abs/1706.03762].
+  Initially, it determines the attention weights by processing Q and K,
+  subsequently combining the outcomes using K. Throughout this function, we
+  utilize the following uppercase letters to represent specific parameters of
+  JTensor:
```

Comment:
What's a JTensor? Perhaps you meant "array"?

Reply:
Done.